docs/develop/make-office-web-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Make your Office Web Add-in compatible with an existing COM add-in-in
+description: 'Enable compatibility with an equivalent COM add-in that has the same functionality as your Office web add-in'
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# Make your Office Web Add-in compatible with an existing COM add-in
+
+If you have an existing COM add-in, you can build equivalent functionality in your Office web add-in to extend your solution features to other platforms such as online or macOS. However, Office web add-ins don't have all of the functionality available in COM add-ins. Your COM add-in may provide a better experience than the web add-in on Windows in Excel, Word, and PowerPoint.
+
+You can configure your Office web add-in so that when an equivalent COM add-in is already installed on the user's computer, Office runs the COM add-in instead of your web add-in. The COM add-in is called "equivalent" because Office will seamlessly transition between the COM add-in and the web add-in depending on which is installed on Windows.
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+## Specify an equivalent COM add-in in the manifest
+
+To enable compatibility with an existing COM add-in, identify the equivalent COM add-in in the manifest of your web add-in. Then Office will use the COM add-in instead of your web add-in when running on Windows.
```

Comment:
```suggestion
To enable compatibility with an existing COM add-in, identify the equivalent COM add-in in the manifest of your Office Add-in. Then Office will use the COM add-in instead of your web add-in when running on Windows.
```